common/js/auth-buttons.js

```diff
@@ -46,4 +50,55 @@
     registration.searchParams.set('returnToUrl', location)
     location.href = registration
   }
-})(solid)
+
+  async function getAccountInfo () {
+    const SOLID = $rdf.Namespace('http://www.w3.org/ns/solid/terms#')
+
+    // load data
+    const accountUrl = new URL('/account', location).href
+    const accountStore = $rdf.graph()
+    const accountFetcher = new $rdf.Fetcher(accountStore)
+    await accountFetcher.load(accountUrl)
+
+    // get DOM and data-dependencies
+    const quotaNode = accountStore.any($rdf.sym(accountUrl), SOLID('storageQuota'))
+    const usageNode = accountStore.any($rdf.sym(accountUrl), SOLID('storageUsage'))
+    const storagePanel = document.getElementById('StoragePanel')
+    const storageQuota = document.getElementById('StorageQuota')
+    const storageUsage = document.getElementById('StorageUsage')
+    const storageUsedBar = document.getElementById('StorageUsedBar')
+    if (!quotaNode || !usageNode || !storagePanel || !storageQuota || !storageUsage || !storageUsedBar) {
+      // If anything of this is not available, we do not need to do anything more
+      return
+    }
+
+    // do stuff
```

Comment:
Not quite a fan of such comments, they don't really add any value... :-)

Reply:
Fixed in commit ee1a116fca8c569051465432bfae1ffdb911811a ^_^